bindings/python/triplestore/triplestore.py

```diff
@@ -0,0 +1,608 @@
+'''A module encapsulating different triplestores using the strategy design
+pattern.
```

Comment:
The level of self-documentation seems uneven. Here there is a lot of information at the top of the file, but interface.py has none...

Reply:
That is because this file provides the interface that the users are expected to see. The other files are internal to the package and are therefore much less documented.

But we could consider to move the module docstring from triplestore.py to the main __init__.py file if you think that would make more sense.